docs/contributing/rebrand-feature-flag.md

```diff
@@ -0,0 +1,35 @@
+# The 'rebrand' feature flag
+
+> [!NOTE]
+> This documentation is for the GOV.UK Design System team and is not relevant to users wanting to contribute to GOV.UK Frontend or the GOV.UK Design System
```

Comment:
I wonder if we need to separate out things like release docs and this which are for the team, and, like, browser support and application architecture, which are relevant to folks trying to do contributions. Not necessarily something for this commit though.

Reply:
It's tricky because a lot of the stuff that applies to us in 'contributing' also applies to anyone trying to do contribution. In theory we're an open source project so anyone can contribute to us in any form, so it's all relevant info for using the codebase.

I think I'm actually gonna remove this. It feels a bit gratuitous looking at it again. I think I put it here because this is very opinionated documentation but that's not a reason to make a big song and dance.